### PR TITLE
SSH tunneling how-to guide reference example

### PR DESCRIPTION
**Pull request by jim-wang-intel**
Closes: https://github.com/edgexfoundry/edgex-docs/issues/134

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [X] Other... Please describe:

This contains a reference example for SSH tunneling running from docker environment for device-virtual.

## What is the current behavior?

Issue Number: https://github.com/edgexfoundry/edgex-docs/issues/134

## What is the new behavior?
A reference example of ssh tunneling implemented in docker

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information
Move reference example artifacts from edgex/Docs into this repo area

### CODE REVIEW

File: ssh-tunneling/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Reference Implementaton for SSH Tunneling on EdgeX Device Services
+
+This is a reference implementation for using SSH Tunneling on EdgeX device-virtual services.
+
+## Build and Run
+
+Please see the detailed steps in "how-to guide" for SSH Tunneling in documentation repository here: <https://github.com/jim-wang-intel/edgex-docs/blob/device-ssh-tunneling-how-to/docs_src/microservices/security/Ch-SSH-Tunneling-HowToSecureDeviceServices.md#put-it-all-together>.
```
Comment (bnevis-i) [inline ssh-tunneling/README.md:7]: Update the URL to point at the final destination.

File: ssh-tunneling/Dockerfile-primary-ds-proxy [hunk 2]
```diff
@@ -0,0 +1,25 @@
+FROM alpine:latest
```
Comment (bnevis-i) [inline ssh-tunneling/Dockerfile-primary-ds-proxy:1]: Add copyright, license.

File: ssh-tunneling/Dockerfile-remote-sshd [hunk 3]
```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:16.04
```
Comment (bnevis-i) [inline ssh-tunneling/Dockerfile-remote-sshd:1]: Add copyright, license.

File: ssh-tunneling/Vagrantfile [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# -*- mode: ruby -*-
```
Comment (bnevis-i) [inline ssh-tunneling/Vagrantfile:1]: Add copyright, license.

File: ssh-tunneling/build.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (bnevis-i) [inline ssh-tunneling/build.sh:1]: Add copyright, license.

File: ssh-tunneling/README.md (hunk 1 above)
Comment (jim-wang-intel) [reply ssh-tunneling/README.md:7]: ok, noted

**Comment by jim-wang-intel** [comment]
close as this had been done

**Pull request closed without merging**